inlong-sdk/transform-sdk/src/main/java/org/apache/inlong/sdk/transform/process/function/ConvFunction.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.sdk.transform.process.function;
+
+import org.apache.inlong.sdk.transform.decode.SourceData;
+import org.apache.inlong.sdk.transform.process.Context;
+import org.apache.inlong.sdk.transform.process.operator.OperatorTools;
+import org.apache.inlong.sdk.transform.process.parser.ValueParser;
+
+import lombok.extern.slf4j.Slf4j;
+import net.sf.jsqlparser.expression.Expression;
+import net.sf.jsqlparser.expression.Function;
+
+import java.math.BigInteger;
+import java.util.List;
+
+/**
+ * ConvFunction  ->  conv(N,from_base,to_base)
+ * description: Converts numbers between different number bases. The minimum base is 2 and the maximum base is 36.
+ *              If from_base is a negative number, N is regarded as a signed number. Otherwise, N is treated as
+ *              unsigned. This function works with 64-bit precision.
+ * - returns NULL if any of its arguments are NULL.
+ * - returns conversion results otherwise.
+ */
+@Slf4j
+@TransformFunction(names = {"conv"})
```

Comment:
Please change the function name to radix_convert

Reply:
done.